pandevice/network.py

```diff
@@ -1099,30 +1099,29 @@ class AggregateInterface(PhysicalInterface):
                 * layer3
                 * layer2
                 * virtual-wire
-                * tap
                 * ha
-                * decrypt-mirror
-                * aggregate-group
 
             Not all modes apply to all interface types (Default: layer3)
 
         ip (tuple): Layer3: Interface IPv4 addresses
         ipv6_enabled (bool): Layer3: IPv6 Enabled (requires
             IPv6Address child object)
-        management_profile (ManagementProfile): Layer3: Interface Management
-            Profile
+        management_profile (ManagementProfile): Layer3: Interface Management Profile
         mtu(int): Layer3: MTU for interface
         adjust_tcp_mss (bool): Layer3: Adjust TCP MSS
         netflow_profile (NetflowProfile): Netflow profile
-        lldp_enabled (bool): Layer2: Enable LLDP
-        lldp_profile (str): Layer2: Reference to an lldp profile
-        netflow_profile_l2 (NetflowProfile): Netflow profile
+        lldp_enabled (bool): Enable LLDP
+        lldp_profile (str): Reference to an lldp profile
         comment (str): The interface's comment
-        ipv4_mss_adjust(int): TCP MSS adjustment for ipv4
-        ipv6_mss_adjust(int): TCP MSS adjustment for ipv6
+        ipv4_mss_adjust(int): Layer3: TCP MSS adjustment for ipv4
+        ipv6_mss_adjust(int): Layer3: TCP MSS adjustment for ipv6
         enable_dhcp (bool): Enable DHCP on this interface
-        create_dhcp_default_route (bool): Create default route pointing to default gateway provided by server
-        dhcp_default_route_metric (int): Metric for the DHCP default route
+        create_dhcp_default_route (bool): Layer3: Create default route pointing to default gateway provided by server
+        dhcp_default_route_metric (int): Layer3: Metric for the DHCP default route
+        enable_lacp (bool): Enables LACP
```

Comment:
Change this to `lacp_enable` to match what the code below is naming the variable.